ci_deployer/deploy.sh

```diff
@@ -4,6 +4,9 @@ git clone https://github.com/Open-IoT-Service-Platform/platform-launcher.git -b
 cd platform-launcher
 echo y | make update
 helm init --upgrade --wait
+export DOCKERUSER=${DOCKER_USERNAME}
+export DOCKERPASS=${DOCKER_PASSWORD}
+export NODOCKERLOGIN=true
```

Comment:
This is used to skip credential verification, which would require to install more tools on the deployer container.